src/Process/LayeredMaterialNodeProcessing.js

```diff
@@ -178,21 +178,25 @@ export function updateLayeredMaterialNodeImagery(context, layer, node) {
             initNodeImageryTexturesFromParent(node, node.parent, layer);
         }
     }
-    // An update is pending / or impossible -> abort
-    if (!node.layerUpdateState[layer.id].canTryUpdate(ts)) {
-        return;
-    }
 
     // Node is hidden, no need to update it
     if (!node.isDisplayed() || !layer.visible) {
         return;
     }
 
+    // TODO: move this to defineLayerProperty() declaration
+    // to avoid mixing layer's network updates and layer's params
```

Comment:
wait, isn't it already the case in View.js:174?

Reply:
Ok, done for geometryLayer, but not for color layers.

Reply:
Actually, this code set an implicit dependency on LayeredMaterial or at least part of its api (more or less logical for this update function here). So if we move it somewhere else, the dependency will also be transferred. I'm not against it per se, but it's something we want to keep it mind.